x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/support/mapper/ExpressionRoleMapping.java

```diff
@@ -139,11 +138,7 @@ public ExpressionRoleMapping(StreamInput in) throws IOException {
         this.name = in.readString();
         this.enabled = in.readBoolean();
         this.roles = in.readStringCollectionAsList();
-        if (in.getTransportVersion().onOrAfter(TransportVersions.V_7_2_0)) {
```

Comment:
Leaked into the wrong PR?

Reply:
Yes, but eeeeh, it'll be going anyway...